### ISSUE TITLE
formatIso formating :30 or :45 timezones incorrectly

### ISSUE THREAD

**Issue by GyllieGyllie** [date-fns/date-fns#1640]
When you use the formatIso function and are in a timezone that is not nicely an hour of but for examle xx:30 or xx:45 the hour in the formatted string is written as .5+30

Example:
2020-02-19T00:00:00+5.5:30

This is causing our backend validators to flag these strings as invalid ISO8601 as it really should look like:
2020-02-17T00:00:00+05:30

Also am using the latest version of date-fns

**Comment by Imballinst** [comment]
I think it should be fixed by https://github.com/date-fns/date-fns/pull/1599 but it's not released yet -- @kossnocorp do you have any idea on when that PR is going live?

Alternatively, if you need it urgently, I think you can install from the [latest master's commit](https://docs.npmjs.com/cli/install):

```
npm install <githubname>/<githubrepo>[#<commit-ish>]

# or
yarn add <githubname>/<githubrepo>[#<commit-ish>]
```

**Comment by GyllieGyllie** [comment]
Thanks for the info, for now we wrote a temporary fix in our own code to transform it as needed so we'll wait for the new release before we update

**Comment by kossnocorp** [comment]
I was overwhelmed lately, thank you for your patience! The fix was released with `2.10.0`.
